pkg/sensors/program/program.go

```diff
@@ -206,6 +206,7 @@ func (p *Program) Unload() error {
 	}
 	p.unloader = nil
 	p.unloaderOverride = nil
+	p.LoadedMapsInfo = nil
```

Comment:
mmh just wait before merging, if the pinned is still there, this might be wrong to do that, I'll should check if ref are to 0 before considering that the maps have been removed.

Reply:
it should be good now, I just noticed that `--keep-sensors-on-exit` break disabling policies, but that's another issue.